packages/devtools_app/lib/src/service/resolved_uri_manager.dart

```diff
@@ -0,0 +1,51 @@
+import '../shared/globals.dart';
+import 'vm_service_wrapper.dart';
+
+/// Manager for handling package Uri lookup and caching.
+class ResolvedUriManager {
+  Map<String, String?>? _resolvedUrlMap;
+  VmServiceWrapper? service;
```

Comment:
initialize and null out the service in vmServiceOpened and vmServiceClosed. then use that service instead of serviceManager.service below

Reply:
oops I don't know how this snuck in here. it isn't used so I will delete it all together :D